api/v1alpha1/scaledobject_types.go

```diff
@@ -12,6 +12,8 @@ import (
 // +kubebuilder:printcolumn:name="ScaleTargetKind",type="string",JSONPath=".status.scaleTargetKind"
 // +kubebuilder:printcolumn:name="ScaleTargetName",type="string",JSONPath=".spec.scaleTargetRef.name"
 // +kubebuilder:printcolumn:name="Triggers",type="string",JSONPath=".spec.triggers[*].type"
+// +kubebuilder:printcolumn:name="Min Replicas",type="integer",JSONPath=".spec.minReplicaCount"
```

Comment:
Do we maybe want to use just Min and Max as the labels to avoid a lot of empty space in the output?

Reply:
> I'd go for `MIN REPLICAS | MAX REPLICAS` - OK for you @zroubalik ?
> 
> _Originally posted by @tomkerkhove in https://github.com/kedacore/keda/issues/1410#issuecomment-766797579_

This suggestion is by @tomkerkhove is this naming causing the error? @coderanger

Reply:
Min/Max is also fine for me

Reply:
should I change the names from MIN REPLICAS | MAX REPLICAS to MIN | MAX?
@coderanger @tomkerkhove

Reply:
WDYT @zroubalik ?

Reply:
Yeah, `Min/Max` is fine by me